packages/vm/Cargo.toml

```diff
@@ -29,10 +29,6 @@ cranelift = ["wasmer/cranelift"]
 # See https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options
 bench = false
 
-[[example]]
-name = "check_contract"
-required-features = ["iterator"]
```

Comment:
iterator became a default feature now, okay. But what happens if this is compiled without defaults and without iterator? Is the example still working?

Reply:
First, I tried `$ cb --no-default-features --example check_contract` and it compiled.

iterator is now a default feature of `check_contract`, by the way. So, it became a runtime setting. I tried 
```
$ ../../target/debug/examples/check_contract --supported-features "" ./testdata/hackatom.wasm 
Supported features: {}
contract checks passed.
$ 
```
and even that worked.

So, this looks like some obsolete stuff to me?

Reply:
You are right though that if compiled without iterator support, other contracts fail. So, perhaps is better to keep this as-is. Will revert.

Reply:
👍